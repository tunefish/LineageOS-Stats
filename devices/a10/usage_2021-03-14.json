{
  "16.0": 42,
  "17.1": 725,
  "18.1": 179,
  "none": 235
}