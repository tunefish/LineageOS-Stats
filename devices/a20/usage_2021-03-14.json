{
  "17.1": 2419,
  "18.1": 108,
  "none": 24
}