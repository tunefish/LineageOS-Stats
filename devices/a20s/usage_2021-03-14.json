{
  "17.1": 2479,
  "none": 1
}