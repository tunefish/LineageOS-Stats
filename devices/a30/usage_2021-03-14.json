{
  "17.1": 29,
  "18.0": 699,
  "18.1": 315,
  "none": 4
}