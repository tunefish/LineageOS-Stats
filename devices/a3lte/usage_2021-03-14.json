{
  "15.1": 654,
  "16.0": 233,
  "17.1": 755
}