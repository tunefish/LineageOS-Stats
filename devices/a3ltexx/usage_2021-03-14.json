{
  "14.1": 1017
}