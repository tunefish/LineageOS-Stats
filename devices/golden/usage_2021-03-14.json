{
  "13.0": 1154,
  "14.1": 3884,
  "15.1": 4,
  "17.1": 7,
  "none": 1
}